hooks/post/30-upgrade.rb

```diff
@@ -180,30 +85,8 @@ def fail_and_exit(message)
     if Kafo::Helpers.module_enabled?(@kafo, 'katello')
       upgrade_step :db_seed, :run_always => true
 
-      upgrade_step :correct_repositories, :long_running => true, :run_always => true
-      upgrade_step :correct_puppet_environments, :long_running => true, :run_always => true
-      upgrade_step :clean_backend_objects, :long_running => true, :run_always => true
-
-      upgrade_step :import_package_groups, :long_running => true
-      upgrade_step :import_rpms, :long_running => true
-      upgrade_step :import_distributions, :long_running => true
-      upgrade_step :import_puppet_modules, :long_running => true
-      upgrade_step :import_subscriptions, :long_running => true
-      upgrade_step :import_product_content, :long_running => true
-      upgrade_step :add_export_distributor, :long_running => true
-      upgrade_step :remove_docker_v1_content, :long_running => true
-      upgrade_step :update_puppet_repository_distributors, :long_running => true
-      upgrade_step :update_subscription_facet_backend_data, :long_running => true
       upgrade_step :remove_event_queue
```

Comment:
Why do you leave this in?

Reply:
@ekohl I don't think there were any upgrade rake tasks added for 3.8, so we technically will only require them to be at 3.7 at a minimum, which seems fairly safe to do now.    Most of these were dropped, but all the :always_run ones were moved to katello here: https://github.com/Katello/katello/pull/7504

I left the remove_event_queue because it is not a rake task (and not really related to any of my work).  I don't mind removing it if its not needed anymore though.

Reply:
I'd be in favor of removing it as well. IMHO `db_seed` should also be removed. The installer should take care of this and if it doesn't that's a bug. Given the new rake task will hard fail on this, we'll be automatically notified by this in our pipelines.

Reply:
done!